editoast/Dockerfile

```diff
@@ -27,7 +27,8 @@ FROM base_builder AS test_builder
 RUN rustup component add llvm-tools && \
     rustup component add rustfmt && \
     rustup component add clippy && \
-    cargo install grcov
+    cargo install --locked grcov && \
+    cargo install --locked taplo-cli
```

Comment:
I don't think it's still needed

Reply:
Indeed, thanks for catching that. Fixed in [c3e350b](https://github.com/osrd-project/osrd/pull/7053/commits/c3e350b24ade6ede3c5714c86414f6bfcbdbfe35).